arm-software/linux/CHANGELOG.md

```diff
@@ -0,0 +1,33 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+## [20.1.0]
+
+This is the first release of the Arm Toolchain for Linux (ATfL), a successor of
+the Arm Compiler for Linux.
+
+Although it is entirely based on the LLVM project version 20.1, there were some
+changes introduced specifically for the ATfL. Most notably, these were:
+
+- The compiler uses a config file by default, which improves
+  performance-specific optimizations; most notably, it encourages the use of the
+  vectorized math routines in the Loop Vectorizer which enables the possibility
+  of vectorizing loops containing the calls to the mathematical library
```

Comment:
Argh. I thought I wrote a comment here about "math" ans "mathematical". Somehow, I can't find it in the "comments". It isn't important, but it feels a bit jarring. Can we either use math or mathematical in both of the above two places?

Reply:
Done